bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/FileInputImpl.java

```diff
@@ -56,6 +63,10 @@ public class FileInputImpl extends AbstractFieldImpl implements FileInput {
     @Default(values = FileInput.DEFAULT_BUTTON_TEXT)
     protected String buttonText;
 
+    public String getFileInputDragDropText() {
```

Comment:
will this be part of the json form schema? If not then use @JsonIgnore

Reply:
Added @JsonIgnore